log.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"os"
+	"regexp"
+)
+
+const (
+	// Severity.
+
+	// From /usr/include/sys/syslog.h.
+	// These are the same on Linux, BSD, and OS X.
+	LOG_EMERG int = iota
+	LOG_ALERT
+	LOG_CRIT
+	LOG_ERR
+	LOG_WARNING
+	LOG_NOTICE
+	LOG_INFO
+	LOG_DEBUG
+)
+
+const (
+	EnvLogLevel      = "LOG_LEVEL"
+	EnvLogFormat     = "LOG_FORMAT"
+	defaultLogFormat = "<%s>\t%s" // #1 severity, #2 msg
```

Comment:
Sorry, I don't get it. The code (documentation not at all) relies on stdout,stderr OR syslog. Journald should be another option since there should be log fields as key value pairs, could be any data and should not be part of this PR.

The log needs then more preparation - see logrus journald hook: https://github.com/wercker/journalhook/blob/master/journalhook.go

The tab relies on the difference log level text width on stdout/stderr. The log format can still be overwritten with this environment variable - thats the reason its made for.